README.md

```diff
@@ -1,2 +1,35 @@
-# hw01
-Repository for first CFSS homework assignment
+# HW 01 #
+<hr />
+## Short biography (*of sorts*) ##
+<hr />
+ * **Name** 
+          <br /> &nbsp; &nbsp;&nbsp;&nbsp; Esha Banerjee 
+ * **Motto**
+            <br /> &nbsp; &nbsp;&nbsp;&nbsp; "I love sleep. My life has the tendency to fall apart when I’m awake, you know?" <small> -Ernest Hemingway </small>
+ * **Hobbies** 
+     * Giving opinions (..everyone has one :wink:)
+     * Trekking cause mountains are :heart:
+     * Cooking *(Fattening others improves self-esteem)* 
+     * Collecting trivia
+     <br /> &nbsp;&nbsp;*UChicago fun fact:* The university logo in use today was introduced in Fall 2012. Here's the old logo:
+<img src ="https://tse1.mm.bing.net/th?id=OIP.Ma33bcd64a5a7e9f57c11396825f87dafo0&pid=15.1&P=0&w=300&h=300" width = "48">
+     
+     
+*Degree*, *skillsets* and *work experience* are best suited for resumes,so if you would really like to know, check out my [LinkedIn profile](https://www.linkedin.com/in/esha-banerjee-29a4854b?trk=nav_responsive_tab_profile_pic)!
+
+If random rants and comments are what interest you, find me on [Twitter](https://twitter.com/eshabanerjee).
+
+For a detailed peek into my life, look me up on [Facebook](https://www.facebook.com/esha.yinyang).
+
+#### Editing README.md ####
```

Comment:
You don't need the hashtags at the end of the line. `#### Editing README.md` works just fine